ruby-monday-blog/spec/features/member_can_navigate_to_post_from_index_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.feature "member can navigate to post from post index", type: :feature do
+  let!(:post) { Post.create(title: 'My Post', body: 'My Content') }
+
+  scenario "user visits post index and clicks on a post headline" do
+    # log_in_as user
+    visit posts_path
+    expect(page).to have_css "a", text: 'My Post'
```

Comment:
As @matugm mentioned, we could put a custom `has_link` matcher, here. I'm a bit ambivalent on this subject, as the `have_css 'a'` is already fairly terse, and I'm wary about adding/creating DSL.  Let me know if you want this and I can implement it.